gp-includes/routes/project.php

```diff
@@ -398,7 +395,7 @@ public function permissions_post( $project_path ) {
 			if ( $this->invalid_and_redirect( $new_permission, gp_url_current() ) ) {
 				return;
 			}
-			$permission = GP::$validator_permission->create( $new_permission );
+			$permission          = GP::$validator_permission->create( $new_permission );
```

Comment:
This should stay as is I think since the next line is a condition.

Reply:
Sure, I thinh phpcs is forcing this because of the ternary operator format.
Another false positive I guess

Reply:
This will keep being signaled by phpcs, I'll separate with a line to break the phpcs need for alignment.